hw/top_earlgrey/dv/chip_sim_cfg.hjson

```diff
@@ -188,6 +188,12 @@
       sw_images: ["sw/device/tests/flash_ctrl_test:1"]
       en_run_modes: ["sw_test_mode"]
     }
+    {
+      name: chip_sw_otp_ctrl_hw_cfg
```

Comment:
Please see chip testplan - this is a LC ctrl test, so this should be `chip_sw_lc_ctrl_hw_cfg`. The corresponding OTP ctrl entry in the tesplan cross references this. 

The easy way to think about it is where exactly is the check happening - in this case, the OTP ctrl only produces the value - its the LC ctrl CSRs that are checking it. 

In addition, please update the testplan so that both testplan entries are marked with this test.